### ISSUE TITLE
Add missing alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [pollysiegel/colmar-academy#1]
For the rest of the subsequent images used in your project, you start the alt attribute but omit the actual description of your image.

The alt attribute is used by "screen reader" software so that a person who is listening to the content of a webpage (for instance, a person who is blind) can interact with this element. Every image should have an alt attribute to be accessible, but it need not contain text. It can be an empty or null attribute: `alt=""`.

https://github.com/pollysiegel/colmar-academy/blob/f824f4f1aa5b28fe4661da4f2241e47736a5b8ce/index.html#L66

**Comment by pollysiegel** [comment]
I did this deliberately; to me, none of the images really had "content" in them that had semantic meaning to someone using a screen reader; they were mostly in there for visual interest. (Perhaps the "picture of Oliver Sacks" is one exception.)  I was under the impression from what I'd read that it was better to include a null alt tag in those cases.

**Comment by jim-rosales** [comment]
Well, it seems like the null attribute is `alt=""` at least that's what [W3C](https://www.w3.org/TR/WCAG20-TECHS/H67.html) shows. If you did that deliberately then it's perfectly fine although from the reference from W3C it is still best practice to have the `=""`

**Comment by pollysiegel** [comment]
Yeah, I agree. I've fixed the tagging.  (Somewhere sometime I thought that I'd seen someone use the alt tag in that way, but I agree with your take on it.  It's odd that my IDE didn't give me warning.)
